python/paddle/v2/parameters.py

```diff
@@ -42,9 +43,26 @@ def create(layers):
 
 class Parameters(object):
     """
-    Parameters is a dictionary contains Paddle's parameter. The key of
-    Parameters is the name of parameter. The value of Parameters is a plain
-    :code:`numpy.ndarry` .
+    `Parameters` manages all the learnable parameters in a neural network.
+    It stores parameters' information in an OrderedDict, key of which is
```

Comment:
key of which is => key is ..., and value is ...

Reply:
done.